src/2.0/schema/schema.json

```diff
@@ -142,7 +147,12 @@
               "properties": {
                 "type": {
                   "type": "string",
-                  "enum": ["NONE"]
+                  "enum": ["NONE", "REVOCATION_STORE"]
+                },
+                "address": {
+                  "type": "string",
+                  "pattern": "^0x[a-fA-F0-9]{40}$",
+                  "description": "Smart contract address for Revocation Store type revocation"
```

Comment:
help me understand this revocationStore property here, what is the difference between this and the revocation object itself?
@yehjxraymond

Reply:
Do you mean between the `revocation` key  and the "REVOCATION_STORE" value ?

Reply:
not really, this is the generated type for v2 schema for the type `Issuer`:
```ts
export interface Issuer {
    /**
     * Issuer's id, DID can be used
     */
    id?:            string;
    identityProof?: IdentityProof;
    /**
     * Issuer's name
     */
    name:        string;
    revocation?: Revocation;
    /**
     * Smart contract address of token registry
     */
    tokenRegistry?: string;
    /**
     * Smart contract address of document store
     */
    documentStore?: string;
    /**
     * Smart contract address of certificate store. Same as documentStore
     */
    certificateStore?: string;
    revocationStore?:  RevocationStore;
}
```
note that theres already a `revocation` key and it is exactly the same (value-wise) as the `revocationStore` key.

so was wondering why is there a need for the `revocationStore` key in the first place.

Reply:
![image](https://user-images.githubusercontent.com/7406870/107457853-cedbbc80-6b8d-11eb-85b4-e4fb1a2bbc0b.png)

Think that the original consideration was that the revocation block will tell us what to use and then in the case of `REVOCATION_STORE` we find the revocationStore block. Additionally, seemed appropriate to have it at the same level at the `documentStore` and `certificateStore`

Seem like if we generalise the address->location, it should be able to support any type of revocation within that block. 

I'll suggest we remove `revocationStore`... v2 is quite fucked anyway lol

Reply:
got it